src/main/java/net/william278/huskhomes/gui/config/Settings.java

```diff
@@ -20,9 +20,12 @@ public class Settings {
     private String language = "en-gb";
     @YamlComment("Options for the home/warp list menu GUI")
     @YamlKey("menu.rows")
-    private int menuSize;
+    private int menuSize = 4;
     @YamlKey("menu.show_controls")
-    private boolean showMenuControls;
+    private boolean showMenuControls = true;
+    @YamlKey("menu.display-operation-help-in-item")
```

Comment:
```suggestion
    @YamlKey("menu.display_controls_help_in_lore")
```
keys should use underscores rather than dashes if possible